apps/emqx_utils/test/emqx_utils_json_SUITE.erl

```diff
@@ -111,7 +113,7 @@ t_decode_encode(_) ->
         <<"int">> => 10,
         <<"foo">> => <<"bar">>
     },
-    ?assertEqual(JsonText, encode({decode(JsonText, [])})),
```

Comment:
When reviewing, please ensure I did everything correctly in the JSON encoder, as I had to modify one existing test to make it pass. 

In the original test, the decoded value is wrapped in a tuple before encoding. If this is intentional, I have to modify my PR to support this scenario. However, I assume the test is supposed to check if `decode`->`encode` will yield the same text.